sql/database.sql

```diff
@@ -9529,7 +9529,7 @@ CREATE TABLE `form_eye_mag_prefs` (
   `selection` varchar(255) DEFAULT NULL,
   `ZONE_ORDER` int(11) DEFAULT NULL,
   `GOVALUE` varchar(10) DEFAULT '0',
-  `ordering` tinyint(4) DEFAULT NULL,
+  `ordering` smallint(4) DEFAULT NULL,
```

Comment:
hi @ShubhamBansal1997 , Should it have the `(4)` part after it and/or should it be different?
(rec reading up on what that number is)

Reply:
Reason for putting up ` 4` is the `2048` basically the `4` is the size here of the column, although the range of smallint is `-32768 to 32767` but 4 stops entry of anything greater than the length(not numeric size) of `4`. Do we keep it `6` I keep it 4 because of `2048`

Reply:
hi @ShubhamBansal1997 , Check out his article on this subject (btw, I also thought the same as you until I read the article).

Reply:
hi @bradymiller , which article?

Reply:
lol, that would be helpful; here it is:
https://thisinterestsme.com/mysql-numbers-in-brackets/